src/styles/mixins.scss

```diff
@@ -0,0 +1,17 @@
+@mixin square {
+  width: $length;
+  height: $side;
+}
+
+@mixin day-box {
+  font-family: Arial, sans-serif;
+  font-size: 30px;
+  display: flex;
+  justify-content: center;
+  align-items: center;
+  position: absolute;
+  left: 0;
+  right: 0;
+  top: 0;
+  bottom: 0;
```

Comment:
I tried to leave a comment for here but apparently i deleted by mistake sorry for that. 

In this part i couldn't understand the necessity of position absolute which then you use it in the `&__day.` When i remove it there happens to be a larger gap in between pieces, so i tried to recommend justify content and align items for `.page`

Reply:
The reason I used absolute positioning on the `::before` is so that the element fits the entire space of the box, I used `position: relative;` on `&__day` so that each `::before` element is relative to its box